lib/sync/util.js

```diff
@@ -38,21 +38,11 @@ var sortObject = function (object) {
 }
 
 
-var sortedStringify = function (obj) {
-  var str = '';
-  try {
-    if (obj) {
-      str = JSON.stringify(sortObject(obj));
-    }
-  } catch (e) {
-    doLog(SYNC_LOGGER, 'error', 'Error stringifying sorted object:' + e);
```

Comment:
@paolobueno @wei-lee Thinking about the `JSON.stringify` example here again (I just happened to be passing the same code again in my editor), how useful is the try/catch really, given that we're just going to re-throw the error anyway?

AFAICT, all we get from keeping is an additional error log message saying `Error stringifying sorted object: [object Object]`. At least I think that's how it will come out, given that it *is* an object and not null or undefined or another type -- none of those seem to cause an error to be thrown. Are there other inputs for which it would throw an error?